libs/langchain-azure-cosmosdb/src/azure_cosmosdb_nosql.ts

```diff
@@ -68,7 +68,7 @@ export interface AzureCosmosDBNoSQLInitOptions {
  */
 export interface AzureCosmosDBNoSQLConfig
   extends AzureCosmosDBNoSQLInitOptions {
-  readonly client?: CosmosClient;
+  client?: CosmosClient;
```

Comment:
The client should not be overridable by the user aside from the constructor, please keep it read-only. When creating the new client in the semantic cache, you can use the constructor to use it.